tardis_env3.yml

```diff
@@ -59,7 +60,6 @@ dependencies:
   - pytest-html
   - pytest-cov
   - coverage
-  - requests
```

Comment:
What did requests do that is no longer needed?

Reply:
It's already on line 25